chart/templates/_jobs.tpl

```diff
@@ -2,26 +2,26 @@
 ==========================================================================================
 | _jobs.update $
 +-----------------------------------------------------------------------------------------
+| * Move some jobs from their manifest locations to other instance groups.
 | * Create $.Values.jobs and $.Values.addon_jobs from the deployment manifest.
 | * Define CC workers based on their property settings in both "jobs" and "resources" trees.
-| * Move some jobs from their manifest locations to other instance groups.
 ==========================================================================================
 */}}
 {{- define "_jobs.update" }}
+  {{- /* *** Move some jobs to new instance groups *** */}}
+  {{- range $from_ig, $ig := $.Values.move_jobs }}
```

Comment:
Given the things we've been doing, I wonder if this needs to be under `$.Values` at all (rather than something in `$.kubecf` that isn't modifiable by the user).

Probably not this PR, though.

Do please update the comment in [`chart/config/jobs.yaml`](https://github.com/cloudfoundry-incubator/kubecf/blob/7762c65756fde37a126f1f5e701bb6056c93f55a/chart/config/jobs.yaml#L6) to drop the reference to `qmove`; with this change I'm pretty sure we want to get _away_ from doing that.

Reply:
I was originally hoping that you could move jobs into new instance groups during `helm install` time. I have a second commit that gets us 70%, but it adds a lot of complexity and templating logic for no real benefit, so I abandoned it.

So yes, the data could live somewhere else, but we only have a mechanism to load `config/*.yaml` files into `$.Values`, so it would require additional setup, just to hide the data from an **internal** config file, so I don't see the point.

I did remove the `qmove` reference; you are correct, we should no longer need the qcopy/qmove ops at all, but do the options by manipulating the manifest in memory, the way this PR already implements it.